frontend/packages/dev-console/src/components/pipelineruns/logs/pipelineRunLogSnippet.ts

```diff
@@ -60,7 +60,7 @@ export const getLogSnippet = (pipelineRun: PipelineRun): PipelineRunErrorDetails
   }
 
   const containerName = failedTaskRun.status.steps?.find(
-    (step: PLRTaskRunStep) => step.terminated.exitCode !== 0,
+    (step: PLRTaskRunStep) => step.terminated?.exitCode !== 0,
```

Comment:
Good catch... Wish TS warned that this failed the type 😕

Reply:
We need to finetune eslint for this, but will have too much errors then.. :thinking: 

It crashs locally and adding this here is a workaround to creating a separate Jira, BZ and PR for this one char :smile: